Web/Admin.UI/src/app/components/tenant/acquisition-log/acquisition-log-view/acquisition-log-view.component.scss

```
@@ -91,6 +92,15 @@ $skipped-status: #ffbf00;
   border-radius: 0.375rem;
 }
 
+.status-inoperable {
+  @extend .log-status;
```

Stylelint (17.3.0)

[error] Expected a placeholder selector (e.g. %placeholder) to be used in `@extend` (scss/at-extend-no-missing-placeholder)

(scss/at-extend-no-missing-placeholder)